From: Yang Li <yang.lee@linux.alibaba.com>
Subject: lib: objpool: fix some kernel-doc comments
Date: Wed, 9 Nov 2022 11:57:04 +0800

Make the description of @oh to @head in objpool_pop()
to clear the below warnings:

lib/objpool.c:425: warning: Excess function parameter 'oh' description in 'objpool_pop'
lib/objpool.c:425: warning: Function parameter or member 'head' not described in 'objpool_pop'

Link: https://bugzilla.openanolis.cn/show_bug.cgi?id=2821
Link: https://lkml.kernel.org/r/20221109035704.88975-1-yang.lee@linux.alibaba.com
Signed-off-by: Yang Li <yang.lee@linux.alibaba.com>
Reported-by: Abaci Robot <abaci@linux.alibaba.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/objpool.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/objpool.c~lib-objpool-added-ring-array-based-lockless-mpmc-queue-fix
+++ a/lib/objpool.c
@@ -414,7 +414,7 @@ static inline void *__objpool_try_get_sl
  * objpool_pop: allocate an object from objects pool
  *
  * args:
- * @oh:  object pool
+ * @head:  object pool used to allocate an object
  *
  * return:
  *   object: NULL if failed (object pool is empty)
_
